.circleci/config.yml

```diff
@@ -24,7 +24,7 @@ jobs:
   build:
     working_directory: /mnt/ramdisk/project
     docker:
-      - image: cimg/node:lts
+      - image: cimg/node:16.17
```

Comment:
Did this in reference to @P0lip's pr to get ci to pass
https://github.com/stoplightio/elements/pull/2274